### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by fcpranav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,121 @@
...
+This section allows the current features implemented for the application.
+
+### Account Management
+
+The application fully supports login and registration. An account must be created for a user to gain access to the application. Users can navigate to the registration page by clicking "Get Started" on the Onboarding screen. Users with an existing account click "Login" to directly go to the login page. By registering an account, users will automatically be sent a Welcome email.
+
+Privacy Policy:
+
+```
+There is no privacy for your data in this beta application. We will do our best to safeguard your data and follow security practices (we never store/get access to your password, all passwords are salted and hashed), but consider any information you input public information. We will also have the right to send you a welcome email after you sign up!
```
Comment (eanile) [inline README.md]: Should probably remove this. If not, format should be adjusted as when viewing the markdown its all on one line

File: README.md [hunk 2]
```diff
@@ -1,37 +1,121 @@
...
+```
+
+### Browse Recipes
+
+Users can browse a maximum of 20 recipes at a time. The recipes shown on initial load are a random selection and not an indication of a small recipe database. At this point in time, pagination is not supported.
```
Comment (eanile) [inline README.md:25]: We should add pagination to the future improvements

File: README.md [hunk 3]
```diff
@@ -1,37 +1,121 @@
...
+
+#### Search Functionality
+
+Users can narrow down specific recipes using the search bar at the top. The search functionality is a unified search of (1) ingredient search, (2) recipe name search, (3) tags search. At this time, only single ingredient search is supported.
```
Comment (eanile) [inline README.md:29]: You may want to add a note about the grocery cart here. I know you mention it next, but just a brief note is good to offset the "only single ingredient"

Also, might want to add multi ingredient search to future improvements.

File: README.md [hunk 4]
```diff
@@ -1,37 +1,121 @@
...
+
+Users can further narrow down their searchs by applying specific filters. The current supported filters are: (1) Filter by cost, (2) Filter by cooking time, (3) Filter by grocery cart. 
+
+### Recipe Details
+
+Users can click specific recipes from the browsing page to get more details on the recipe. Through a horizontal slider, users can see the description of the recipe, the ingredients required, and a list of instructions for that recipe. 
+
+#### Step-By-Step Instructions
+
+Users can press the "Let's Go" button to start the cooking process and have steps prompted to them while they cook. At this time, data on cooking times for each step is missing. Each step currently has a default time of 10 seconds to showcase the feature. Users can pause (and resume) a given step to work at their own pace. For each step, images for required equipment + ingredients is shown as a slideshow.
```
Comment (eanile) [inline README.md:41]: We should add that for future improvements, showing the actual time remaining along with the increasing button thing

File: README.md [hunk 5]
```diff
@@ -1,37 +1,121 @@
...
+
+Through the Drawer Navigation, users can navigate to their personalized grocery cart and shopping list.
+
+#### Grocery Cart
+
+A list users can use to reflect what groceries they currently own. The grocery cart directly interacts with the "Filter by grocery cart".
+
+#### Shopping List
+
+A list users can use to reflect what ingredients they need to buy on their next shopping trip. Once users buy their entire list, they can use the "Move to Grocery Cart" functinality to move all items to their grocery cart.
```
Comment (eanile) [inline README.md]: functionality typo

File: README.md [hunk 6]
```diff
@@ -1,37 +1,121 @@
...
+
+### Future Improvements
```
Comment (eanile) [inline README.md:55]: Refer to previous comments about potential other improvements

File: README.md [hunk 7]
```diff
@@ -1,37 +1,121 @@
...
+
+#### Client Application
+
+Navigate to `./PlateUp-reactnative`.
+
+Install all package dependencies using `npm install`.
+
+***Without Backend Server Running***
+
+There is an option to run the application using stubbed data. We use MirageJS to make this possible. See `mock-http.js` to see what HTTP requests are stubbed out. You may need to add additional ones. To use stubbed data, see `App.js` and uncomment the code `mockHTTP()`.
```
Comment (eanile) [inline README.md]: Should be updated, as we don't uncomment anything but instead set a flag to true.

**Review by eanile — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (fcpranav) [reply README.md:29]: This is specifically for search functionality so I think it's fine not to include it here. I'll add it to future improvements.

File: README.md (hunk 7 above)
Comment (fcpranav) [reply README.md]: Nice catch.

File: README.md [hunk 8]
```diff
@@ -52,13 +50,15 @@ A list users can use to reflect what groceries they currently own. The grocery c
...
 
 ### Future Improvements
 
-1. Step-By-Step Instructions - Populate real cooking time data
+1. Step-By-Step Instructions - Populate real cooking time data and add a numeric timer
 2. Shopping List / Grocery Cart - Support automatic unit conversions
 3. Currently, the grocery cart is mostly manually updated (other than the shopping list flush). The API already supports an endpoint that allows users to "check" whether they meet a recipe's ingredients requirements. If the user meets the ingredients, ingredients will automatically be deducted from their grocery cart. If the user does not meet the ingredients, missing ingredients will be added to their shopping list. The mobile application does not support this feature at the moment and this can be an improvement for the next release.
+4. Search bar improvements - Support multiple ingredients search, currently it is limited by one ingredient at a time
+5. Browse recipes - Add pagination support so that a user can brwose more than 20 recipes at a time
```
Comment (eanile) [inline README.md]: browse typo

**Review by eanile — CHANGES_REQUESTED** [review]

**Review by eanile — APPROVED** [review]

**Pull request merged**